source/use_cases/aws-restaurant-management-demo/README.md

```diff
@@ -42,17 +64,33 @@ This demo implements three REST APIs using AWS Gateway, one for each user type.
   - `orderId` - the unique order id.
 
 ### Kitchen staff
-- `/list-open-orders` - GET - lists all orders that have a status of 'OPEN'.
+- `/get-open-orders` - GET - lists all orders that have a status of 'OPEN'.
 - `/complete-order` - POST - updates the status of an order to 'FILLED'. Request parameters:
   - `orderId` - the unique order id.
 
 ### Managers
-- `/list-orders` - GET - lists all orders, regardless of status.
-- `/close-out-service` - POST - triggers the close-out process, invoking a Step Functions workflow that generates a 
+- `/get-all-orders` - GET - lists all orders, regardless of status.
+- `/close-out-service` - ANY - triggers the close-out process, invoking a Step Functions workflow that generates a 
   sales report, calculates tips for all servers, and archives orders from DynamoDB to Redshift.
-- `/reports` - POST - get a specific sales report from the static assets bucket.  
+- `/get-report` - POST - get a specific sales report from the static assets bucket.  
   - `filename` - the filename of the report.
 
+## Using the API
+This demo provisions an architecture that is ready to be hooked up to a front-end application of your choice/design. 
+Since the three API endpoints provisioned by this architecture are secured by Cognito, it is recommended to review the
+steps listed [here](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-invoke-api-integrated-with-cognito-user-pool.html) 
```

Comment:
This link doesn't get it done - they have to write an Amplify app. We need to tell them how to do it with the CLI so they can invoke the API with curl or postman.